src/main/java/sirius/kernel/nls/NLS.java

```diff
@@ -340,9 +340,9 @@ public static Optional<String> getIfExists(String property, @Nullable String lan
             return Optional.empty();
         }
         String targetLanguage = Strings.isEmpty(lang) ? getCurrentLang() : lang;
-        if (translation.hasTranslation(targetLanguage)) {
-            return Optional.of(translation.translate(Strings.isEmpty(lang) ? getCurrentLang() : lang,
-                                                     getFallbackLanguage()));
+        String fallbackLanguage = getFallbackLanguage();
```

Comment:
maybe only fetch this if needed (split into two checks) - as otherwise we perform a threadlocal lookup for no reason (which is super fast - but super unneccessarry in this case :-) )

Reply:
done